event-buffer/event-buffer-liquibase/src/main/resources/liquibase/event-buffer-changesets/004-add-component-to-stream-buffer.changelog.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet
+            id="event-buffer-004"
+            author="TechPod"
+            logicalFilePath="004-add-component-to-stream-buffer.changelog.xml">
+
+        <dropPrimaryKey
+                constraintName="stream_buffer_pkey"
+                tableName="stream_buffer"
+        />
+        <renameColumn
+                oldColumnName="version"
+                newColumnName="position"
+                tableName="stream_buffer"
+        />
+
+        <addColumn
+                tableName="stream_buffer">
+            <column
+                    name="component"
+                    type="varchar(100)"
+                    defaultValue="event_listener">
```

Comment:
Should this be upper case or ?

Reply:
updated to EVENT_LISTENER in latest push